### ISSUE TITLE
New ZIM: Wikivoyage Thailand

### ISSUE THREAD

**Issue by Popolechien** [openzim/zim-requests#229]
In light of the upcoming Wikimania, it would be nice to have an (English) Wikivoyage custom app for Thailand.

As a first step, a simple nopic ZIM would be needed, but is there any way we can generate a tiled landing page like for the other Wikipedia subsets?

**Comment by kelson42** [comment]
@Popolechien I'm not against the idea, but I would prefer to release the few custom apps we have in the pipe before starting a new one.

**Comment by Popolechien** [comment]
Yep, I'm opening the ticket but realistically I would not release (and announce) before May or June.

**Comment by RavanJAltaie** [comment]
@Popolechien What's the update in that? as we don't have a link or files?

**Comment by kelson42** [comment]
Nothing has been done AFAIK and Wikimania will not take place in Thailand, right?

**Comment by Popolechien** [comment]
Yeah it's Singapore now. I still think the idea is good tbh.
